base/applications/notepad/lang/he-IL.rc

```diff
@@ -155,17 +155,17 @@ BEGIN
     STRING_UNTITLED "ללא שם"
     STRING_ALL_FILES "כל הקבצים (*.*)"
     STRING_TEXT_FILES_TXT "קבצי טקסט (*.txt)"
-    STRING_TOOLARGE "File '%s' is too large for notepad.\n\
-Please use a different editor."
-    STRING_NOTEXT "You didn't enter any text. \
-\nPlease type something and try again"
-    STRING_DOESNOTEXIST "File '%s'\ndoes not exist\n\n\
-Do you want to create a new file ?"
-    STRING_NOTSAVED "File '%s'\nhas been modified\n\n\
-Would you like to save the changes ?"
+    STRING_TOOLARGE "הקובץ '%s' גדול מידי עבור פנקס הרשימות\n\
+בבקשה השתמש בעורך טקסט אחר."
+    STRING_NOTEXT "לא הקלדת טקסט. \
```

Comment:
```suggestion
    STRING_NOTEXT "לא הקלדת טקסט.\
```

Reply:
^ this one could be fixed later, all translations are affected: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=STRING_NOTEXT

Reply:
Yes, I do concur. Can be done in a separate PR also.